### PR TITLE
[Fix] Change the type of `label` from `int32` to `int64` in the args of `F.cross_entropy` for pytorch 1.11

### PR DESCRIPTION
**Pull request by lpxgis**
**Bug Fixes**
In Pytorch 1.11, `F.cross_entropy` only allows the labels with the type of `int64` rather than `int32`. We fix the bug in the `get_target()` function of `roi_embed_head.py`.

### CODE REVIEW

**Review by GT9505 — COMMENTED** [review]
There is a lint error in the CI. Please follow [this](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to install pre-commit hook before you perform "git commit" operation.

**Review by GT9505 — APPROVED** [review]
